core/pager.rs

```diff
@@ -21,18 +27,21 @@ const PAGE_UPTODATE: usize = 0b001;
 const PAGE_LOCKED: usize = 0b010;
 /// Page had an I/O error.
 const PAGE_ERROR: usize = 0b100;
+/// Page had an I/O error.
```

Comment:
comment I believe is not correct right? here is page is not aligned with written content?